<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryObjectWidget</class>
 <widget class="QWidget" name="CategoryObjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QPushButton" name="btnCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">[icon]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblChannels">
     <property name="text">
      <string>%n channel(s)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblViewers">
     <property name="text">
      <string>%n viewer(s)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
